src/tokens.js

```diff
@@ -2089,6 +2089,56 @@ module.exports = [
             },
         ],
     },
+    {
+        name: 'Scrim',
+        description: 'Values for transparent light and dark curtains that cover content.',
+        tokens: [
+            {
+                id: 'scrim-light-80',
+                type: 'string',
+                platforms: {
+                    scss: {
+                        name: '$tp-scrim__light-80',
+                        value: 'rgba(255, 255, 255, 0.8)',
+                    },
+                    javascript: {
+                        name: 'tpScrimLight80',
+                        value: 'rgba(255, 255, 255, 0.8)',
+                    },
+                    ios: {
+                        name: 'light-80',
```

Comment:
The `-` isn't valid in Swift. I think this should be `light80`.

Reply:
Done